src/unfold/decorators.py

```diff
@@ -37,16 +55,16 @@ def inner(model_admin, request, *args, **kwargs):
 
 
 def display(
-    function=None,
+    function: Optional[Callable] = None,
```

Comment:
you can also specify parameters of this and following callable

we should know what `@display` annotated function should accept as parameters